Mimir 1849 endre punktum til desimaltegn tooltip engelske highchart

Carl-OW opened the pull request:
Fix for mimir-1849 og mimir-1851

johnnadeluy reviewed (changes requested):
I suggest only including the translated phrases in the highchart-lang-en file to avoid duplication, and to then merge the two into a new json object when defining the english phrases. (See comment)
Alternatively, if the default language for highcharts is english then we can consider dropping the highcharts-lang-en.json file and only set accessibility options when the chart has to be translated to norwegian.
Also, don't forget to apply the changes to the Highmap and nameSearch components as well.

Carl-OW commented:
I have deleted highmap-lang-en.json and opted for another solution, where the default highcharts language is used for for the en version of the site and the accessibilityLang file is used for nb version, I also added the fix for nameSearch, so that the decimalPoints for the graphs are correct for each language, as well as the download-dropdown text

Carl-OW reviewed (commented):
updated code fix, and changed nameSearch.ts to return "nb", instead of "no"

johnnadeluy reviewed (approved):
Looks good! Good job 🌟 You could probably also get rid of the commented out code before merge 😊
